api/spec/controllers/mno_enterprise/jpi/v1/marketplace_controller_spec.rb

```diff
@@ -70,108 +70,85 @@ def hash_for_apps(apps)
     describe 'GET #index' do
       subject { get :index }
 
-      context 'when marketplace_listing is set' do
+      before do
+        api_stub_for(get: '/apps', response: from_api([app]))
+        # TODO: Shouldn't need to stub this
+        api_stub_for(get: "/apps/#{app.id}/shared_entities", response: from_api([]))
```

Comment:
I don't worry too much about this as this is being removed as part of the switch to api v2